src/core/lib/surface/call_log_batch.cc

```diff
@@ -108,8 +108,8 @@ char* grpc_op_string(const grpc_op* op) {
 }
 
 void grpc_call_log_batch(const char* file, int line, gpr_log_severity severity,
-                         grpc_call* call, const grpc_op* ops, size_t nops,
-                         void* tag) {
+                         grpc_call* /*call*/, const grpc_op* ops, size_t nops,
```

Comment:
Similar comment here: If these params are unused, we should probably just remove them.